v2.12.x/tools-v2.12/mtrace-reader.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python3
```

Comment:
@kv2019i is it simple to change to sof-mtrace-reader.py ?

Reply:
@lgirdwood Not anymore for this release as this is tagged and would need to be changed in SOF repo. I'd really prefer to drop these tools altogether from sof-bin, but I've been including them as they've been included in the past. 

Maybe I'll just stop now? I can just stop including the tools into the release tarball, and stop including these in future pull requests and that's it basicly.